nodejs/week3/homework/warmup/index.js

```diff
@@ -0,0 +1,56 @@
+const express = require("express");
+const app = express();
+
+app.use(express.urlencoded({extended: true}));
+app.use(express.json()) // To parse the incoming requests with JSON payloads
+
+app.get("/", (request, res) => res.send("nodejs week3 homework"));
+
+app.get("/calculator/add", (request, response) => {
+  console.log(request.query);
+  const firstParam = parseInt(request.query.firstParam);
+  const secondParam = parseInt(request.query.secondParam);
+  const sum = firstParam + secondParam;
```

Comment:
what if you had more parameters as in the homework's example?
Could you use the reduce function?
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/reduce